plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import math
+import os
+from typing import Any
+
+from omegaconf import DictConfig, OmegaConf
+
+from hydra.core.plugins import Plugins
+from hydra.plugins.sweeper import Sweeper
+
+# noinspection PyUnresolvedReferences
+from hydra.test_utils.test_utils import TSweepRunner, sweep_runner  # noqa: F401
+from hydra_plugins.hydra_ax_sweeper import AxSweeper
+
+
+def test_discovery() -> None:
+    """
+    Tests that this plugin can be discovered via the plugins subsystem when looking for Sweeper
+    :return:
+    """
+    assert AxSweeper.__name__ in [x.__name__ for x in Plugins.discover(Sweeper)]
+
+
+def quadratic(cfg: DictConfig) -> Any:
+    x = cfg.quadratic.x
+    a = 100
+    b = 1
+    z = a * (x ** 2) + b
+    return z
+
+
+def test_launched_jobs(sweep_runner: TSweepRunner,) -> None:  # noqa: F811
```

Comment:
Add additional test covering:
1. command line only usage (no config).
2. mixed command line and config usage.